src/Reference/HasOneSql.php

```diff
@@ -54,12 +54,15 @@ function (Model $ourModel) use ($theirFieldName) {
                     return $ourModel->refLink($this->link)->action('field', [$theirFieldName])->reset('order');
                 },
             ],
-            $defaults
+            $defaults,
```

Comment:
Am I right  that there are no functional changes in HasOneSql class? Only some prettifying.

Reply:
Only one major change in `HasOneSql::ref`. The below code is now executed irrespective if ourModel loaded or not (originally was only when not loaded)
```
$ourFieldExpression = $ourModel->action('field', [$ourField]);

return $theirModel->addCondition($theirField, $ourFieldExpression);
```